install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1408,7 +1408,7 @@ except this time, specify the `MODE=refresh` option:
 +
 ----
 $ oc new-app --as=system:serviceaccount:openshift-infra:metrics-deployer \
-    -f metrics-deployer.yaml \
+    -f /usr/share/ansible/openshift-ansible/roles/openshift_hosted_templates/files/v1.4/enterprise/metrics-deployer.yaml \
```

Comment:
@ahardin-rh I think I may have mentioned the other day that the Origin ifdef didn't matter since this was on the enterprise-3.4 branch, but I forgot we build Origin 1.4 docs using enterprise-3.4:

https://docs.openshift.org/1.4/welcome/index.html

So this could also include an ifdef'd for origin as:

````
-f ~/openshift-ansible/roles/openshift_hosted_templates/files/v1.4/origin/metrics-deployer.yaml
````

Reply:
@adellape updated! 🎆